tests/java/simple-command-injection-direct-input-java-test.yml

```
@@ -0,0 +1,59 @@
+id: simple-command-injection-direct-input-java
+valid:
+  - |
+    @GetMapping("/run/{command}")
+    public ResponseEntity<?> run1(
+        @PathVariable final String command
+    ) {
+        ResponseEntity<?> response = ResponseEntity.noContent().build();
+        try {
+            String foo = command + "something something...";
+            Runtime.getRuntime().exec(foo); 
```

yamllint

[error] trailing spaces

(trailing-spaces)